testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -660,6 +660,30 @@ QUnit.module('interaction via swipe', {
 
         assert.equal($multiView.find('.dx-state-focused').length, 0, 'there is no focused class on any item');
     });
+
+    QUnit.test('selected index should be set to the second item after left swipe from the first', function(assert) {
```

Comment:
should we add ticket's ID to the name of tests?

Reply:
No